### PR TITLE
User is able to see the password using the eye button

### PR DESCRIPTION
**Pull request by agatsoh**

### CODE REVIEW

File: resources/templates/account_password.html [hunk 1]
```diff
@@ -9,9 +9,13 @@
 <div class="info-panel">
   <div>
     <input type="password" name="passphrase1" placeholder="Secure password" />
+    <a>
+      <img id="eye" class="icon-link" src="{{ static_url('images/eye.svg') }}" />
+    </a>
   </div>
   <div>
     <input type="password" name="passphrase2" placeholder="Please confirm" />
+    &emsp;
```
Comment (manuelwedler) [inline resources/templates/account_password.html:18]: Why is that needed?

Comment (agatsoh) [reply resources/templates/account_password.html:18]: As i remember it now since I dont put `a ` tag (of the eye img) for the second input i require a tiny amount of spacing there otherwise the inputs get misaligned

Comment (manuelwedler) [reply resources/templates/account_password.html:18]: Now I see, thanks!
